prow/plugins/cla/cla.go

```diff
@@ -174,3 +182,79 @@ func handle(gc gitHubClient, log *logrus.Entry, se github.StatusEvent) error {
 	}
 	return nil
 }
+
+func handleCommentEvent(pc plugins.PluginClient, ce github.ReviewCommentEvent) error {
+	return handleComment(pc.GitHubClient, pc.Logger, &ce)
+}
+
+func handleComment(gc gitHubClient, log *logrus.Entry, e *github.ReviewCommentEvent) error {
+	// Only consider open PRs and new comments.
+	if e.PullRequest.State != "open" || e.Action != github.ReviewCommentActionCreated {
+		return nil
+	}
+	// If we create a "/check-cla comment, check for the cla in commit statuses, add cla label if necessary.
+	checkCLA := false
+	if checkCLARe.MatchString(e.Comment.Body) {
+		checkCLA = true
+	} else {
+		return nil
+	}
+
+	org := e.Repo.Owner.Login
+	repo := e.Repo.Name
+	number := e.PullRequest.Number
+	hasCLAYes := false
+	hasCLANo := false
+
+	labels, err := gc.GetIssueLabels(org, repo, number)
+	if err != nil {
+		log.WithError(err).Errorf("Failed to get the labels on %s/%s#%d.", org, repo, number)
+	}
+	for _, candidate := range labels {
```

Comment:
Seems like for both this loop and the `range statuses` loop we should consider breaking out early and/or better handling the edge cases

Reply:
We could break out if we are absolutely sure that both flags can't simultaneously exist (for some reason).